cruise.umple/test/cruise/umple/implementation/a.rb

```diff
@@ -0,0 +1,40 @@
+# PLEASE DO NOT EDIT THIS CODE
```

Comment:
What is this file doing here? It shouldn't be committed. A recent PR deleted it after testing (as far as i remember)